### PR TITLE
Sstc editorial fixes.

### PR DESCRIPTION
**Pull request by wmat**
The second to last paragraph starts "When STCE in menvcfg is zero, ...".  This should be clarified to say "When this extension is implemented and STCE in menvcfg is zero, ...".  And ditto for the beginning of the last paragraph.

### CODE REVIEW

File: src/sstc.adoc
Comment (aswaterman) [inline src/sstc.adoc:187]: We can undo this last change because menvcfg.STCE cannot be 1 if this extension is not implemented.

Comment (gfavor) [reply src/sstc.adoc:187]: Andrew is correct.  My mild concern is that this leaves an asymmetry between this and the preceding statement - that are saying symmetric things.  Hence creating an implied "exercise for the reader" to realize why there is the asymmetry.
For a concise compromise - that creates a sense of the second statement being a continuation of the first statement - how about starting the second statement "But when STCE in menvcfg is one and STCE in henvcfg is zero, ..."?

Andrew, if you don't like that, then we can stick with dropping the change.  (Although I do also like replacing the 'but' with an 'and' in what is a logical 'A and B' conjunction.)

Comment (gfavor) [reply src/sstc.adoc:187]: Bill, the other two changes look good.

Comment (aswaterman) [reply src/sstc.adoc:187]: I like your proposed alternative, @gfavor.

Comment (gfavor) [reply src/sstc.adoc:187]: Bill, you can go ahead and switch the original change to this modified change.  And then you can merge this whole PR of three changes.

**Review by gfavor — APPROVED** [review]
Assuming that change update, I'm good overall with this PR.

**Review by aswaterman — CHANGES_REQUESTED** [review]
I'll impose a "request changes" just so I get the chance to review the final outcome.

**Comment by gfavor** [comment]
Certainly.  I just wanted to get one approval out of the way so Bill can proceed as soon as you give approval.

File: src/sstc.adoc
Comment (wmat) [reply src/sstc.adoc:187]: I've made this change .

**Review by aswaterman — APPROVED** [review]

**Pull request merged**
